room_capacity/README.md

```diff
@@ -0,0 +1,18 @@
+
+# Auto-calculate rooms needed for a conference 🎟️ in a Google Sheet via an Apps Script
+
+This is a sample to show a simple custom function using an Apps Script in
```

Comment:
Please briefly explain what a custom function is and why someone would like to use one, also adding a link to the docs would be useful

Reply:
Good call, added.